src/Amazon.php

```diff
@@ -74,11 +82,13 @@ class Amazon
      * @param  string $countryCode Country code for Amazon service; may be US, UK, DE, JP, FR, CA
      * @param  string $secretKey   API Secret Key
      * @param  string $version     API Version to use
+     * @param  bool   $useSsl      Use HTTPS instead of HTTP?
      * @throws Exception\InvalidArgumentException
      * @return Amazon
      */
-    public function __construct($appId, $countryCode = 'US', $secretKey = null, $version = null)
-    {
+    public function __construct($appId, $countryCode = 'US', $secretKey = null, $version = null,
```

Comment:
when you make parameters multiline, all of them should be placed on separate lines